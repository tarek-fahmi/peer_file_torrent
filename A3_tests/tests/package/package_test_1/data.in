 Verify file.data Exists
./pkg_main ./valid_1.bpkg -file_check
./pkg_main ./valid_2.bpkg -file_check