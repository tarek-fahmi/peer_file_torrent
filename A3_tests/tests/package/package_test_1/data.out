# Expected output replaces this comment: